openedx/features/edly/tests/test_utils.py

```diff
@@ -238,6 +240,47 @@ def test_user_cannot_login_on_requested_edly_organization(self):
 
         assert user_can_login_on_requested_edly_organization(self.request, self.user) is False
 
+    def test_user_can_login_on_requested_edly_organization_on_login_view(self):
+        """
+        Test user can login on the requested URL site if linked with its parent edly-organization.
+        """
+
+        user_password = 'password'
```

Comment:
can we create set password test helper function? as this is needed in more than one unit tests

Reply:
Done